helm/aad-pod-identity-chart/values.yaml

```diff
@@ -0,0 +1,34 @@
+# Default values for pod-aad-identity-helm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+rbac:
+  enabled: false
```

Comment:
let's make this `true` by default since AKS and acs-engine are both enabled by default now